app/data_providers/snowplow/entities.py

```diff
@@ -1,112 +1,120 @@
 from typing import Dict
 
-from aio_snowplow_tracker import SelfDescribingJson, Subject
+from aio_snowplow_tracker import SelfDescribingJson
 
 from app.data_providers.util import get_dict_without_none
 from app.models.api_client import ApiClient
+from app.models.corpus_recommendation_model import CorpusRecommendationModel
+from app.models.corpus_recommendations_send_event import CorpusRecommendationsSendEvent
 from app.models.corpus_slate_lineup_model import CorpusSlateLineupModel
 from app.models.corpus_slate_model import CorpusSlateModel
-from app.models.unleash_assignment import UnleashAssignmentModel
 from app.models.request_user import RequestUser
+from app.models.unleash_assignment import UnleashAssignmentModel
+
+_CORPUS_RECOMMENDATIONS_SEND_SCHEMA = 'iglu:com.pocket/corpus_recommendations_send/jsonschema/1-0-0'
+_CORPUS_SLATE_SCHEMA = 'iglu:com.pocket/corpus_slate/jsonschema/4-0-0'
+_CORPUS_SLATE_LINEUP_SCHEMA = 'iglu:com.pocket/corpus_slate_lineup/jsonschema/3-0-0'
+_USER_SCHEMA = 'iglu:com.pocket/user/jsonschema/1-0-0'
+_FEATURE_FLAG_SCHEMA = 'iglu:com.pocket/feature_flag/jsonschema/1-0-0'
+_VARIANT_ENROLL_SCHEMA = 'iglu:com.pocket/variant_enroll/jsonschema/1-0-0'
+_API_USER_SCHEMA = 'iglu:com.pocket/api_user/jsonschema/1-0-1'
+
+
+def get_corpus_recommendations_send_event(event: CorpusRecommendationsSendEvent) -> SelfDescribingJson:
+    """
+    :param event:
```

Comment:
nit: I think we should remove comments that don't add value. Comments that repeat the name of the function and parameters clutter the code and make it less readable. I prefer self-documented code + extra comments for unobvious logic for internal code (not a public API).

Reply:
Agreed. I'll follow the convention going forward of only adding docstring when I can add some useful context.